turbo.json

```diff
@@ -2,9 +2,11 @@
   "$schema": "https://turbo.build/schema.json",
   "globalEnv": ["CLOUDFLARE_ACCOUNT_ID", "CLOUDFLARE_API_TOKEN"],
```

Comment:
These env vars should be moved to `env` within `deploy` so that we aren't giving our API token to build scripts when we don't need to.